tests/Fixer/PhpUnit/PhpUnitDataProviderStaticFixerTest.php

```diff
@@ -140,5 +143,25 @@ public function testFoo() {}
     abstract public function provideFooCases();
 }',
         ];
+
+        yield 'fix when containing dynamic calls and with `force` enabled' => [
+            '<?php
+class FooTest extends TestCase {
+    /**
+     * @dataProvider provideFoo1Cases
+     */
+    public function testFoo1() {}
+    public static function provideFoo1Cases() { $this->init(); }
+}',
+            '<?php
+class FooTest extends TestCase {
+    /**
+     * @dataProvider provideFoo1Cases
+     */
+    public function testFoo1() {}
+    public function provideFoo1Cases() { $this->init(); }
+}',
+            ['force' => true],
```

Comment:
let's have test for `force=>false` too , pls

Reply:
done